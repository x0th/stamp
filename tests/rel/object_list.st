Cat = Object^;
Dog = Object^;
Object l = [Cat, Dog];
Object.l
