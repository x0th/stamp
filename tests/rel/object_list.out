STDOUT:
[Cat-hash, Dog-hash]
STDERR:
